### PR TITLE
Renaming

### PR DESCRIPTION
**Pull request by sunbreak1211**

### CODE REVIEW

File: test/Sky.t.sol [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        sky = new Sky();
+  
+        _token_ = address(sky);
+        _contractName_ = "Sky";
+        _tokenName_ ="Sky Governance Token";
+        _symbol_ = "SKY";
+    }
+
+    function invariantMetadata() public view {
+        assertEq(sky.name(), "Sky Governance Token");
```
Comment (telome) [inline test/Sky.t.sol:24]:
```suggestion
        _tokenName_ ="SKY Governance Token";
        _symbol_ = "SKY";
    }

    function invariantMetadata() public view {
        assertEq(sky.name(), "SKY Governance Token");
```

Comment (sunbreak1211) [reply test/Sky.t.sol]: Yeah just realized, fixed!

**Review by telome — APPROVED** [review]
LGTM

**Review by oldchili — APPROVED** [review]
LGTM

**Pull request merged**